News-Android-App/build.gradle

```
@@ -103,22 +114,25 @@ dependencies {
     // implementation 'com.google.android.gms:play-services:4.2.42'
     //implementation project(':Android-SingleSignOn')
     //implementation project(path: ':MaterialShowcaseView:library', configuration: 'default')
-    implementation 'com.github.nextcloud:Android-SingleSignOn:02e651a18b'
+    implementation 'com.github.nextcloud:Android-SingleSignOn:e781a33e01'
     implementation 'com.github.David-Development:MaterialShowcaseView:bf6afa225d'
 
-    implementation "com.android.support:support-v4:${SUPPORT_VERSION}"
-    implementation "com.android.support:support-compat:${SUPPORT_VERSION}"
-    implementation "com.android.support:appcompat-v7:${SUPPORT_VERSION}"
-    implementation "com.android.support:design:${SUPPORT_VERSION}"
-    implementation "com.android.support:palette-v7:${SUPPORT_VERSION}"
-    implementation "com.android.support:recyclerview-v7:${SUPPORT_VERSION}"
-    implementation "com.android.support:customtabs:${SUPPORT_VERSION}"
-    implementation "com.android.support:cardview-v7:${SUPPORT_VERSION}"
+    // https://mvnrepository.com/artifact/androidx.legacy/legacy-support-v4
+    implementation "androidx.legacy:legacy-support-v4:1.0.0"
+    implementation "androidx.core:core:1.1.0-alpha05"
+    implementation "androidx.appcompat:appcompat:1.1.0-alpha03"
```

```suggestion
    implementation "androidx.appcompat:appcompat:1.1.0-alpha02"
```
Downgrading to `alpha02` fixes opening settings with dark theme selected.
Older versions have other issues and `alpha03` causes the settings activity to be relaunched in an infinite loop.